turbo.json

```diff
@@ -143,16 +147,6 @@
         "*.{md,mdx,json,ts,tsx,mjs,yml}"
       ],
       "outputs": ["coverage/**", "junit.xml"]
-    },
-    "test": {
-      "inputs": [
-        "{app,components,hooks,i18n,layouts,middlewares,pages,providers,types,util}/**/*.{ts,tsx,mjs}",
-        "{app,components,layouts,pages,styles}/**/*.css",
-        "{next-data,scripts,i18n}/**/*.{mjs,json}",
-        "{app,pages}/**/*.{mdx,md}",
-        "*.{md,mdx,json,ts,tsx,mjs,yml}"
-      ],
-      "outputs": ["coverage/**", "junit.xml"]
```

Comment:
This appears to be a duplicate of the `test:unit` task so I've removed it, along with the `test` script in `package.json`.